hardhat.config.ts

```diff
@@ -0,0 +1,34 @@
+import { HardhatUserConfig, task } from "hardhat/config";
+import "@nomiclabs/hardhat-etherscan";
+import "@nomiclabs/hardhat-waffle";
+import "@typechain/hardhat";
+import "solidity-coverage";
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
```

Comment:
It's unneeded, so let's remove this sample task

Reply:
Can you please also give a similar opinion on package.json (that is actually the most critical in terms of 'unneeded', because it makes the installation much heavier).

Reply:
I can do even better - ping me when you've finished with the code and I'll open a PR for removing all unneeded clutter 💪